Sources/App/Controllers/PackageController+ShowRoute.swift

```diff
@@ -87,10 +87,10 @@ extension PackageController {
                     return nil
                 }
                 let cl = Link(
-                    label: pluralizedCount(commitCount, singular: "commit"),
+                    label: "commit".pluralizedCount(commitCount),
```

Comment:
I think in case of the interpolation `"\(pluralizedCount: ...)"` I'm happy with with new version but here it's a downside that the count comes after the label. In that respect this is a regression.

Probably not worth spending a lot of time on it but perhaps this could be an extension on `Int` to enable

```swift
commitCount.labelled("commit")
```